ui/src/cloud/utils/reporting.ts

```diff
@@ -31,21 +31,14 @@ export const updateReportingContext = (properties: KeyValue) => {
   reportingTags = {...reportingTags, ...properties}
 }
 
-export const reportEvent = ({
-  timestamp = toNano(Date.now()),
-  measurement,
-  fields,
-  tags,
-}: Point) => {
+const pooledEvent = ({timestamp, measurement, fields, tags}: Point) => {
```

Comment:
calling this var measurement but then changing it to event is super confusing. Can we just call it eventName or whatever throughout ?

Reply:
dig the rename `pooledEvent` 👍

+1 on the suggestion to rename `measurement`